chart.py

```diff
@@ -1,24 +1,34 @@
 # coding=utf-8
 from dateutil.parser import parse
 
+# How to convert from mg/dL (US format) to mmol/L (non-US format)
+MMOLL_CONVERT_FACTOR = 1.0 / 18.0
```

Comment:
Some minor feedback, but I think it'd be best to move the conversion higher up in the execution path. This file is really just munging for the GChart API.

Have you found a way to pull the unit preference from pump settings?